generators/generator-base-private.js

```diff
@@ -871,6 +871,14 @@ module.exports = class extends Generator {
         }
         const mainGeneratorJhipsterVersion = packagejs.version;
         const blueprintJhipsterVersion = blueprintPackageJson.dependencies['generator-jhipster'];
+        if (!blueprintJhipsterVersion) {
+            // Check if generator-jhipster is on peerDependency
```

Comment:
If generator-jhipster is on peerDependency, it most likely won't be declared as dependency, and thus this code will never be executed, since the function will return above (L870)

Reply:
You are faster than me, I would have caught when implementing CI.
I got lost on blueprint, jhipster, yeoman tests.

Thanks